assets/js/blocks/checkout/inner-blocks/checkout-shipping-method-block/shared/helpers.ts

```diff
@@ -2,12 +2,18 @@
  * External dependencies
  */
 import type { CartShippingPackageShippingRate } from '@woocommerce/type-defs/cart';
+import { getSetting } from '@woocommerce/settings';
 
 export interface minMaxPrices {
 	min: CartShippingPackageShippingRate | undefined;
 	max: CartShippingPackageShippingRate | undefined;
 }
 
+const collectibleMethodIds = getSetting< string[] >(
```

Comment:
Nothing wrong here, however, we could probably move this settings call to a new shared util which returns a function called `isRateCollectable` or similar. This can be consumed here and in assets/js/blocks/checkout/inner-blocks/checkout-pickup-options-block/block.tsx

Reply:
If this method accepted either a string or array of strings, you could also replace `areRatesCollectible` with this helper. The code that splits `rate.split( ':' )[ 0 ]` method ids would remain in place. Thoughts?

Reply:
See changes in `assets/js/base/utils/shipping-rates.ts`, this just checks a rate, or rates, and doesn't take care of splitting